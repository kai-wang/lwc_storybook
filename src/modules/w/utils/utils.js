// export { default as _ } from './lodash';
export { clsx } from './clsx';
