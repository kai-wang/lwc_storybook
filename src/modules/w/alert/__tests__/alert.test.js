import { createElement } from 'lwc';
import Alert from 'w/alert';
import {normalizeBoolean} from 'w/utils';

describe('w-alert', () => {
  afterEach(() => {
    // The jsdom instance is shared across test cases in a single file so reset the DOM
    while (document.body.firstChild) {
      document.body.removeChild(document.body.firstChild);
    }
  });

  it('should render', () => {
    // Create initial element
    const element = createElement('w-alert', {
      is: Alert,
    });

    // console.log('template', element.shadowRoot);
    

    return Promise.resolve().then(() => {
      expect(element).toMatchSnapshot();
    });
  });
});
