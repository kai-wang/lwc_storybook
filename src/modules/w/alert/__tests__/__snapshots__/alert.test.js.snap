// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`w-alert should render 1`] = `
<w-alert>
  #shadow-root(open)
</w-alert>
`;
